com.devolutions.remotedesktopmanager.yml

```diff
@@ -0,0 +1,86 @@
+app-id: com.devolutions.remotedesktopmanager
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+sdk-extensions: [org.freedesktop.Sdk.Extension.rust-stable]
+command: remotedesktopmanager
+tags: [proprietary]
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=x11
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  - --share=network
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.Notifications
+  - --filesystem=~/.rdm:create
+  - --filesystem=xdg-run/keyring
+  - --env=SSL_CERT_FILE=/etc/ssl/certs/ca-certificates.crt
+modules:
+  - name: gdk-pixbuf
+    buildsystem: meson
+    config-opts:
+      - -Dothers=enabled # needed for svg
+      - -Dman=false
+      - -Dgtk_doc=false
+      - -Dbuiltin_loaders=all
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/gdk-pixbuf.git
+        tag: 2.42.12
+        x-checker-data:
+          type: git
+          tag-pattern: ^([\d.]+)$
+  - librsvg.json
```

Comment:
the previous comment about having this inline here, and the fact that librsvg is in the runtime still stands.